### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by matevarga** [docker/docker#22657]
**Output of `docker version`:**

```
Client:
 Version:      1.11.0
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   4dc5990
 Built:        Wed Apr 13 18:34:23 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.0
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   4dc5990
 Built:        Wed Apr 13 18:34:23 2016
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
Containers: 55
 Running: 6
 Paused: 0
 Stopped: 49
Images: 66
Server Version: 1.11.0
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 310
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge overlay null host
Kernel Version: 4.2.0-35-generic
Operating System: Ubuntu 14.04.4 LTS
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 7.792 GiB
Name: xxx
ID: 2R52:ZXPR:UD56:KTME:JNOW:2X76:Y6LM:3UCU:4PH6:7ONV:PT6X:ULIL
Docker Root Dir: /var/lib/docker
Debug mode (client): false
Debug mode (server): false
Username: 
Registry: https://index.docker.io/v1/
Cluster store: consul://xxx:8500
Cluster advertise: xxx:2376
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Steps to reproduce the issue:**
Can't reproduce.
**Describe the results you received:**
Docker daemon encountered a panic. Several Docker containers died, I could not restart them (Docker complained about them being `aleady active` (sic).
**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**
Docker stack trace attached.

[docker.txt](https://github.com/docker/docker/files/259096/docker.txt)

**Comment by thaJeztah** [comment]
Thanks for reporting, and for providing the stack trace; this looks to be a duplicate of https://github.com/docker/docker/issues/22081, which is resolved in docker 1.11.1, so upgrading to 1.11.1 should fix this.

I'll close this issue, but thanks again for reporting, and feel free to comment here after I closed

**Comment by matevarga** [comment]
Thanks.
